public/app/sw-register.js

```
@@ -1,107 +1,89 @@
+      if (type === 'start')    statusUI?.setStatus(model, 'loading');
+      if (type === 'done')     statusUI?.setStatus(model, 'cached');
+      if (type === 'cached')   statusUI?.setStatus(model, 'cached');
+      if (type === 'error')    statusUI?.setStatus(model, 'error');
+    };
+  } catch { /* BroadcastChannel unavailable */ }
 
-        case 'cached':
-          if (statusEl) statusEl.textContent = `${d.filename} cached ✓`;
-          break;
+  // Load the blob manifest to get canonical URLs.
+  const manifest = await loadManifest();
```

ESLint

[error] 'manifest' is assigned a value but never used.

(no-unused-vars)